reconcile/blackbox_exporter_endpoint_monitoring.py

```diff
@@ -141,6 +141,14 @@ def get_endpoints() -> dict[EndpointMonitoringProvider, list[Endpoint]]:
     return endpoints
 
 
+def get_blackbox_providers() -> list:
```

Comment:
Any reason not to make this `list[EndpointMonitoringProvider]`? By default it will be `list[Any]` which probably isn't going to help with autocompletion or catching bugs where this is being used?

Reply:
no reason, just an oversight. thanks for pointing it out. fixed in [9a865e5](https://github.com/app-sre/qontract-reconcile/pull/2184/commits/9a865e5920715fd7e5de89cfdb36a1dcb79b2500)